File: apiwrapper/src/main/java/com/pinkertone/apiwrapper/types/UserProfile.java
package com.pinkertone.apiwrapper.types;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;


public class UserProfile {
    @SerializedName("user_data")
    @Expose
    public User user;

    @SerializedName("profile_data")
    @Expose
    public Profile profile;
}

File: apiwrapper/src/main/java/com/pinkertone/apiwrapper/types/UserProfile.kt
package com.pinkertone.apiwrapper.types

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class UserProfile (
    @SerializedName("user_data")
    @Expose
    var user: User? = null,

    @SerializedName("profile_data")
    @Expose
    var profile: Profile? = null,
)